indico/modules/events/abstracts/__init__.py

```diff
@@ -120,15 +120,18 @@ def _get_notification_placeholders(sender, **kwargs):
 def _extend_event_menu(sender, **kwargs):
     from indico.modules.events.abstracts.util import has_user_tracks
     from indico.modules.events.layout.util import MenuEntryData
+    from indico.modules.events.contributions import contribution_settings
 
     def _boa_visible(event):
-        return config.LATEX_ENABLED and event.has_feature('abstracts')
+        return config.LATEX_ENABLED and event.has_feature('abstracts') and contribution_settings.get(event, 'published')
+
 
     def _reviewing_area_visible(event):
         if not session.user or not event.has_feature('abstracts'):
             return False
         return has_user_tracks(event, session.user)
 
+
```

Comment:
I think functions that are not on the top level usually don't have two empty lines in between